mne_realtime/lsl_client.py

```diff
@@ -106,14 +110,15 @@ def _connect(self):
         pylsl = _check_pylsl_installed(strict=True)
         print(f'Looking for LSL stream {self.host}...')
         # resolve_byprop is a bit fragile
-        streams = pylsl.resolve_streams(wait_time=min(0.1, self.wait_max))
+        streams = pylsl.resolve_streams(wait_time=min(1.0, self.wait_max))
         ids = list()
         for stream_info in streams:
```

Comment:
if I'm understanding this correctly then i would also create a separate error for when the `host` results in multiple instances and `host_name` is `None`

something like 
if len(streams) != len(np.unique(streams)) and `host_name` is None:
ValueError('This LSL stream does not have a unique name, please also specify host_name')